### DISCUSSION TITLE
PL-100 & PL200 중급과정 오늘 부터 시작합니까?

### DISCUSSION THREAD

**Discussion by gwh-21c** [fusiondevkr/pl900#74] [1 Q&A]
PL-900 초급과정을 신청 못하고 YOUTUBE 학급하고 있습니다.
초급과정 오리엔테이션시 오늘부터 PL-100 & PL-200 중급과정을 시작하는 것으로 되어 있는데
중급과정에 대한 신청은 벌써 마감이 된 것인지요?

초급과정 정식으로 수료하지 못해도 중급과정 교육에 참여할 수는 없는지요?

**Comment by sesme100** [comment]
안녕하세요. 관심을 가져주셔서 너무나 감사합니다. 
PL-100&200과정은 준비중에 있고 정확한 시작 일자와 접수일자는 확정되어있지 않지만 올해말에 시작을 생각하고 준비하고 있습니다. 그전에 PL-900 기초과정을 먼저 할 예정입니다. 이 과정의 일정도 확정되는 즉시 안내 드리겠습니다. 감사합니다

**Comment by sesme100** [comment]
안녕하세요 PL-900 과정은 11월 15일로 확정되었습니다. PL-100 & 200과정은 내년 2022에 시작할 예정입니다. 

아래의 Forms로 들어가셔서 PL-900과정 신청 주시면 감사하겠습니다!! 
https://forms.office.com/r/CWP6pzJ0NR
